src/main/app/src/app/admin/formulier-definitie-edit/tekstvlak-edit-dialog/tekstvlak-edit-dialog.component.html

```diff
@@ -10,7 +10,12 @@
     subscriptSizing="dynamic"
   >
     <mat-label>{{'defaultWaarde' | translate}}</mat-label>
-    <textarea rows="18" matInput [(ngModel)]="data['value']"></textarea>
+    <textarea
+      rows="18"
+      matInput
+      [ngModel]="data['value']"
+      (ngModelChange)="data['value'] = $event"
```

Comment:
To be sure: these `= $event` additions are part of the Angular upgrade? I don't see ie being mentioned in https://angular.dev/update-guide?v=17.0-18.0&l=3 but maybe that's not the whole story?